vnext/Microsoft.ReactNative/Views/Text/TextVisitor.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <Modules/NativeUIManager.h>
+#include <Modules/PaperUIManagerModule.h>
+#include <UI.Xaml.Documents.h>
+#include <Views/ShadowNodeBase.h>
+
+namespace Microsoft::ReactNative {
+
+class TextVisitor {
+ public:
+  void Visit(ShadowNode *node);
+
+ protected:
+  virtual void VisitExtensionText(ShadowNodeBase *node);
```

Comment:
I find the name slightly confusing - what does it do?

Reply:
Happy to rename - it supports custom view managers. E.g., we're introducing an `IViewManagerCreateWithProperties` in our app that returns a `Hyperlink` derivate of span, so it allows for tree recursion through app custom TextElements.

Reply:
Thought I needed the hook for third-party view managers, but it looks like I probably do not. I can just remove that virtual method and continue recursion for non-core views.